apps/web/src/hooks/infinity/useFarmReward.ts

```diff
@@ -468,24 +471,63 @@ const formatRewardsMap = (data?: Awaited<ReturnType<typeof fetchUserFarmRewards>
   }, {} as Record<string, string>)
 }
 
+const filterActiveRecentRewards = <
+  T extends {
+    campaignId: string
+    epochEndTimestamp: string | Record<string, never>
+  },
+>(
+  rewards: T[] | undefined,
+  activeCampaignIds: string[],
+  timestamp: number,
+): T[] | undefined => {
+  if (!rewards) return undefined
+
+  return rewards.filter((r) => {
+    if (!activeCampaignIds.includes(r.campaignId)) return false
+
+    const epochTs = Number(r.epochEndTimestamp)
+    if (!epochTs || Number.isNaN(epochTs)) return false
+
+    return timestamp - epochTs <= DAY_IN_SECONDS
```

Comment:
if it is active campaign but one day passed without rewards it should be filtered out too